sdk/identity/azure-identity/azure/identity/_internal/msal_managed_identity_client.py

```diff
@@ -23,10 +23,13 @@ class MsalManagedIdentityClient(abc.ABC):  # pylint:disable=client-accepts-api-v
     """Base class for managed identity client wrapping MSAL ManagedIdentityClient."""
 
     # pylint:disable=missing-client-constructor-parameter-credential
-    def __init__(self, **kwargs: Any) -> None:
-        self._settings = kwargs
+    def __init__(
+        self, *, client_id: Optional[str] = None, identity_config: Optional[Mapping[str, str]] = None, **kwargs: Any
+    ) -> None:
+        self._client_id = client_id
+        self._identity_config = identity_config
```

Comment:
_settings is the same concept of option_bag in get_token_info().

We may pass some arguments into msal.ManagedIdentity in the future.

Reply:
Updated to keep the _settings usage